docs/src/man/comparisons.md

```diff
@@ -4,12 +4,30 @@ This section compares DataFrames.jl with other data manipulation frameworks in P
 
 A sample data set can be created using the following code:
 
-```julia
-using DataFrames
-using Statistics
-
-df = DataFrame(grp = repeat(1:2, 3), x = 6:-1:1, y = 4:9, z = [3:7; missing], id = 'a':'f')
-df2 = DataFrame(grp = [1, 3], w = [10, 11])
+```jldoctest
+julia> using DataFrames
```

Comment:
this is not needed.

Reply:
in this section we do not need codes to be runnable. Python/R/Stata codes are not runnable anyway. We just need to make sure that the codes are correct.